app/src/main/java/com/movieapp/dao/MovieDao.java

```diff
@@ -0,0 +1,31 @@
+package com.movieapp.dao;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import com.movieapp.entity.MovieDB;
+
+import java.util.List;
+
+@Dao
+public interface MovieDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    void insert(MovieDB movieDB);
+
+    @Update(onConflict = OnConflictStrategy.REPLACE)
+    void update(MovieDB movieDB);
+
+    @Delete
+    void delete(MovieDB movieDB);
+
+    @Query("SELECT * FROM movie_table ORDER BY vote_average DESC")
```

Comment:
Why you sort movies when you try to do this when you can handle this logic in java or create another Query for this task

Reply:
fixed